highway_env/envs/summon_env.py

```diff
@@ -105,19 +110,33 @@ def _create_vehicles(self):
             Create some new random vehicles of a given type, and add them on the road.
         """
 
-        self.vehicle = Vehicle(self.road, [0, 0], 2*np.pi*self.np_random.rand(), 0)
+        self.vehicle = Vehicle(self.road, self.vehicle_starting, 2*np.pi*self.np_random.rand(), 0)
         self.road.vehicles.append(self.vehicle)
 
-        lane = self.np_random.choice(self.road.network.lanes_list())
-        self.goal = Obstacle(self.road, lane.position(lane.length/2, 0), heading=lane.heading)
+        goalpos = [np.random.choice([-2*self.spots - 10 ,2*self.spots + 10]),0]
+        self.goal = Obstacle(self.road, goalpos, heading=0)
         self.goal.COLLISIONS_ENABLED = False
         self.road.vehicles.insert(0, self.goal)
 
 
         vehicles_type = utils.class_from_path(self.config["other_vehicles_type"])
-        for _ in range(self.config["vehicles_count"]):
-            idx = np.random.randint(0,self.num_middle_lanes)
-            self.road.vehicles.append(vehicles_type.make_on_lane(self.road,("d","e",idx),np.random.randint(0,self.config["vehicles_count"]*10),velocity=-5))
+        for i in range(self.config["vehicles_count"]):
+            coinflip = np.random.rand()
```

Comment:
This is not the best name

Reply:
What would you suggest for better readability?